clustering/cluster_minifigures.py

```diff
@@ -1,34 +1,50 @@
+import json
+
+def parse_content_line(line):
+    if 'Category ID' in line:
+        return False
```

Comment:
```suggestion
        return None
```

to return different types for example String and Boolean is a bad code style.

Therefore I would use None instead of False